### PR TITLE
Add an example workflow WelcomeWorkflow and 2 relevant example tasks that outputs a greeting message.

### PR DESCRIPTION
**Pull request by mayitbeegh**
# TL;DR
Add an example workflow WelcomeWorkflow and 2 relevant example tasks that outputs a greeting message. The example workflow matches the Python example used in https://docs.flyte.org/en/stable/

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Screen Shot 2021-06-25 at 09 30 55
Tested in local sandbox.

## Tracking Issue
_NA_

## Follow-up issue
_NA_

### CODE REVIEW

File: flytekit-examples/src/main/java/org/flyte/examples/AddQuestionTask.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.flyte.flytekit.SdkBindingData;
+import org.flyte.flytekit.SdkRunnableTask;
+import org.flyte.flytekit.SdkTransform;
+import org.flyte.flytekit.jackson.JacksonSdkType;
+
+/**
+ * Example task that takes a greeting message as input, appends "How are you?", and outputs the result
+ */
+@AutoService(SdkRunnableTask.class)
+public class AddQuestionTask extends SdkRunnableTask<AddQuestionTask.Input, AddQuestionTask.Output> {
```
Comment (EngHabu) [inline flytekit-examples/src/main/java/org/flyte/examples/AddQuestionTask.java:14]: Let's heavily comment the code and particularly the public classes/methods... explain why everything the way it's... or what it means to be here... besides giving you the opportunity to learn the different terminology and inner workings, I think it's going to be a useful thing for the customers to understand how to properly write workflows/tasks....

Comment (kumare3) [reply flytekit-examples/src/main/java/org/flyte/examples/AddQuestionTask.java:14]: I agree lets document the code, so that it can serve as a template

Comment (mayitbeegh) [reply flytekit-examples/src/main/java/org/flyte/examples/AddQuestionTask.java]: Added javadoc to the best of my knowledge

**Review by kanterov — APPROVED** [review]

**Comment by kanterov** [comment]
Nice! Thanks

**Pull request merged**